packages/react-reconciler/src/ReactFiberCompleteWork.old.js

```diff
@@ -120,9 +126,11 @@ import {
   prepareToHydrateHostInstance,
   prepareToHydrateHostTextInstance,
   prepareToHydrateHostSuspenseInstance,
+  warnDeleteNextHydratableInstance,
   popHydrationState,
   resetHydrationState,
   getIsHydrating,
+  hasMore,
```

Comment:
This should probably be renamed to something more specific around hydration since "hasMore" can mean a lot in this context.